pkg/sharings/send_mails.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/cozy/cozy-stack/pkg/couchdb"
 	"github.com/cozy/cozy-stack/pkg/instance"
 	"github.com/cozy/cozy-stack/pkg/jobs"
-	"github.com/cozy/cozy-stack/pkg/jobs/workers"
+	"github.com/cozy/cozy-stack/pkg/jobs/workers/mails"
```

Comment:
It's weird indeed, `workers.` works to use mails, but not `mails.`... I changed that with `mails "github.com/cozy/cozy-stack/pkg/jobs/workers/mails"` as a workaround.